body.tex

```diff
@@ -530,7 +612,7 @@ \subsection{Jira Maintenance}
 It is required that the \texttt{Team} field be set to the appropriate team (\jira{RFC-145}).
 This indicates which manager is responsible for \gls{seeing} that the work is completed successfully.
 Available teams, and the associated managers, are listed in the \href{https://developer.lsst.io/processes/jira_agile.html}{Developer Guide}; generally speaking, they align with the the work breakdown structure described in \S\ref{sec:wbs}.
-Where there is uncertainty about which team should be responsible for a particular ticket, the “System Management” team may be used to indicate that the \gls{DM} \gls{Project Manager} is responsible for assigning the work.
+Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{DM} \gls{Project Manager} is responsible for assigning the work.
```

Comment:
I guess there is no DM project manager in ops in charge of Data Production - AD of DP ?

Reply:
```suggestion
Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{AD} of  \gls{Data Production} is responsible for assigning the work.
```